### DISCUSSION TITLE
Control the dpi when taking screenshots in .PNG or .JPEG

### DISCUSSION THREAD

**Discussion by zhengliuX** [pyvista/pyvista#2381] [Q&A] [answered]
Hi, is there any way that we can control the dpi when taking screenshots in PNG or JPEG?

**Comment by stla** [comment, marked as answer]
You can convert to svg with [save_graphic](https://docs.pyvista.org/api/plotting/_autosummary/pyvista.Plotter.save_graphic.html) and use a tool such as Inkscape to convert to png.
